src/main/kotlin/org/rust/ide/intentions/RemoveCurlyBracesIntention.kt

```diff
@@ -9,74 +9,82 @@ import com.intellij.openapi.editor.Editor
 import com.intellij.openapi.project.Project
 import com.intellij.psi.PsiElement
 import org.rust.lang.core.psi.*
-import org.rust.lang.core.psi.ext.ancestorStrict
-import org.rust.lang.core.psi.ext.asTrivial
-import org.rust.lang.core.psi.ext.endOffset
-import org.rust.lang.core.psi.ext.startOffset
+import org.rust.lang.core.psi.ext.*
 
 /**
  * Removes the curly braces on singleton imports, changing from this
  *
  * ```
- * import std::{mem};
+ * use std::{mem};
  * ```
  *
  * to this:
  *
  * ```
- * import std::mem;
+ * use std::mem;
  * ```
  */
-// TODO: this really should reuse code from RsSingleImportRemoveBracesFormatProcessor.
 class RemoveCurlyBracesIntention : RsElementBaseIntentionAction<RemoveCurlyBracesIntention.Context>() {
```

Comment:
Lets also support this case:
```rust
struct A;
mod inner {
    pub struct B;
}
mod foo {
    use crate::{A, inner::B};
    fn foo(b: B) {}
}
```

Expected after invoking "Remove unused import": `use crate::inner::B;`
Actual: `use crate::{inner::B};`

Reply:
Good catch, thanks.